modules/apps/redirect/redirect-service/src/test/java/com/liferay/redirect/internal/util/PatternUtilTest.java

```diff
@@ -68,6 +78,16 @@ public void testInvalidRegexPattern() {
 		PatternUtil.parse(new String[] {"*** a"});
 	}
 
+	@Test
+	public void testSlashPattern() {
```

Comment:
Would it be interesting to have also a test called `testCaretPattern()`?

Reply:
Sure.